src/coreclr/scripts/exploratory.proj

```diff
@@ -0,0 +1,87 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
+
+  <!-- <PropertyGroup>
+     <HelixTargetQueues>Some_Queue</HelixTargetQueues>
+     <Platform>Windows</Platform>
+     <Architecture>x64</Architecture>
+     <RunConfiguration>win-x64</RunConfiguration>
+   </PropertyGroup>
+
+  <Target Name="printItems">
+        <Message Text="@(HelixWorkItem -> 'name: %(HelixWorkItem.Identity)
+     dir: %(HelixWorkItem.PayloadDirectory)
+     pre: %(HelixWorkItem.PreCommands)
+     command: %(HelixWorkItem.Command)
+     post: %(HelixWorkItem.PostCommands)
+     timeout: %(HelixWorkItem.Timeout)  '"/>
+  </Target> -->
+
+  <PropertyGroup Condition="'$(AGENT_OS)' == 'Windows_NT'">
+    <FileSeparatorChar>\</FileSeparatorChar>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(AGENT_OS)' != 'Windows_NT'">
+    <FileSeparatorChar>/</FileSeparatorChar>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(AGENT_OS)' == 'Windows_NT'">
+    <Python>%HELIX_PYTHONPATH%</Python>
+    <OutputDirectory>%HELIX_WORKITEM_UPLOAD_ROOT%</OutputDirectory>
+    <CoreRoot>%HELIX_CORRELATION_PAYLOAD%\CoreRoot</CoreRoot>
+    <ToolPath>%HELIX_CORRELATION_PAYLOAD%\exploratory</ToolPath>
+    <!-- Workaround until https://github.com/dotnet/arcade/pull/6179 is not available -->
```

Comment:
"not" available? https://github.com/dotnet/arcade/pull/6179 was merged a year ago, so can this be cleaned up?

Reply:
This is fixed but I still faced some problems. I will double check.

Reply:
Alright, if i remove that line we see this errors and that's the reason I had it in first place. @safern 

```
D:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\7.0.0-beta.21463.4\tools\download-results\DownloadFromResultsContainer.targets(24,5): error MSB4018: The "DownloadFromResultsContainer" task failed unexpectedly. [D:\workspace\_work\1\s\src\coreclr\scripts\exploratory.proj]
D:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\7.0.0-beta.21463.4\tools\download-results\DownloadFromResultsContainer.targets(24,5): error MSB4018: System.IO.IOException: The filename, directory name, or volume label syntax is incorrect. : 'D:\workspace\_work\1\s\src\coreclr\scripts\$([MSBuild]::NormalizeDirectory('$(BUILD_SOURCESDIRECTORY)', 'artifacts', helixresults'))\9e889cb3-8222-4f97-b8c4-c95ff9d8fa88' [D:\workspace\_work\1\s\src\coreclr\scripts\exploratory.proj]
D:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\7.0.0-beta.21463.4\tools\download-results\DownloadFromResultsContainer.targets(24,5): error MSB4018:    at System.IO.FileSystem.CreateDirectory(String fullPath, Byte[] securityDescriptor) [D:\workspace\_work\1\s\src\coreclr\scripts\exploratory.proj]
D:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\7.0.0-beta.21463.4\tools\download-results\DownloadFromResultsContainer.targets(24,5): error MSB4018:    at System.IO.Directory.CreateDirectory(String path) [D:\workspace\_work\1\s\src\coreclr\scripts\exploratory.proj]
D:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\7.0.0-beta.21463.4\tools\download-results\DownloadFromResultsContainer.targets(24,5): error MSB4018:    at Microsoft.DotNet.Helix.Sdk.DownloadFromResultsContainer.ExecuteCore() in /_/src/Microsoft.DotNet.Helix/Sdk/DownloadFromResultsContainer.cs:line 66 [D:\workspace\_work\1\s\src\coreclr\scripts\exploratory.proj]
D:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\7.0.0-beta.21463.4\tools\download-results\DownloadFromResultsContainer.targets(24,5): error MSB4018:    at Microsoft.DotNet.Helix.Sdk.DownloadFromResultsContainer.Execute() in /_/src/Microsoft.DotNet.Helix/Sdk/DownloadFromResultsContainer.cs:line 58 [D:\workspace\_work\1\s\src\coreclr\scripts\exploratory.proj]
D:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\7.0.0-beta.21463.4\tools\download-results\DownloadFromResultsContainer.targets(24,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\workspace\_work\1\s\src\coreclr\scripts\exploratory.proj]
D:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\7.0.0-beta.21463.4\tools\download-results\DownloadFromResultsContainer.targets(24,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [D:\workspace\_work\1\s\src\coreclr\scripts\exploratory.proj]
##[error].packages\microsoft.dotnet.helix.sdk\7.0.0-beta.21463.4\tools\download-results\DownloadFromResultsContainer.targets(24,5): error MSB4018: (NETCORE_ENGINEERING_TELEMETRY=Build) The "DownloadFromResultsContainer" task failed unexpectedly.
System.IO.IOException: The filename, directory name, or volume label syntax is incorrect. : 'D:\workspace\_work\1\s\src\coreclr\scripts\$([MSBuild]::NormalizeDirectory('$(BUILD_SOURCESDIRECTORY)', 'artifacts', helixresults'))\9e889cb3-8222-4f97-b8c4-c95ff9d8fa88'
   at System.IO.FileSystem.CreateDirectory(String fullPath, Byte[] securityDescriptor)
   at System.IO.Directory.CreateDirectory(String path)
   at Microsoft.DotNet.Helix.Sdk.DownloadFromResultsContainer.ExecuteCore() in /_/src/Microsoft.DotNet.Helix/Sdk/DownloadFromResultsContainer.cs:line 66
   at Microsoft.DotNet.Helix.Sdk.DownloadFromResultsContainer.Execute() in /_/src/Microsoft.DotNet.Helix/Sdk/DownloadFromResultsContainer.cs:line 58
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask)

```

Reply:
It seems like helixresults in that call to `NormalizeDirectory` is missing an opening `'`?

Reply:
will you be able to send a fix? I won't block this PR, but will send follow-up PR to validate and remove this.

Reply:
Would you be able to validate that fixes the issue locally? You can update the targets locally from the nuget cache and then send the job from your local box and see if it works?

Reply:
I was not able to verify this on local machine and probably take me time, but looking at other references (e.g. https://github.com/dotnet/runtime/blob/2e08554621bcd8415bccd581e7be06b48c7813d8/src/mono/wasm/build/WasmApp.targets#L167), I think `$(BUILD_SOURCESDIRECTORY)` doesn't need the surrounding `'`'. I will send a PR to arcade.

Reply:
Sent out https://github.com/dotnet/arcade/pull/7961